Sources/Apollo/ApolloClient.swift

```diff
@@ -28,8 +28,9 @@ public typealias OperationResultHandler<Operation: GraphQLOperation> = (_ result
 
 /// The `ApolloClient` class provides the core API for Apollo. This API provides methods to fetch and watch queries, and to perform mutations.
 public class ApolloClient {
-  let networkTransport: NetworkTransport
-  let store: ApolloStore
+  public let networkTransport: NetworkTransport
```

Comment:
I'd prefer not to allow users to change these objects after client creation. Could we make this `private(set) public` instead?